src/components/EditableHeading/EditableHeading.tsx

```diff
@@ -1,244 +1,166 @@
-import { camelCase } from "lodash-es";
-import { getStyle } from "../../helpers/typesciptCssModulesHelper";
 import { ComponentDefaultTestId, getTestId } from "../../tests/test-ids-utils";
```

Comment:
I renamed the old component to `LegacyEditableHeading` so the new one gets the old one's name